### DISCUSSION TITLE
Resistance gained from affliction is additive which makes it quite easy to reach damage immunity

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14403] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

damage resistance from afflictions is additive one can easily reach 100% damage resistance
vigor: 50% resistance at 600%
damageresistance (moloch gene): 25% resistance at 100%
stonewall (security talent): 25% resistance

### Reproduction steps

1. get all afflictions mentioned above
2. take damage
3. notice that you can only get bleeding

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
do note character can also get resistances from CharacterAbilityModifyResistance and equipments

**Comment by Regalis11** [comment]
This is intentional
